plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -55,15 +76,43 @@ impl FromStr for CgroupV2Metric {
 
 #[derive(Clone)]
 pub struct Metrics {
```

Comment:
Here the name of the structure is okay :)
Apply my remark about the renaming here too.